mm2src/mm2_number/src/mm_number.rs

```diff
@@ -73,24 +71,19 @@ impl<'de> Deserialize<'de> for MmNumber {
     where
         D: Deserializer<'de>,
     {
-        let raw: Box<RawValue> = Deserialize::deserialize(deserializer)?;
-
-        if let Ok(dec) = BigDecimal::from_str(raw.get().trim_matches('"')) {
-            return Ok(MmNumber(from_dec_to_ratio(&dec)));
-        };
```

Comment:
Got it. I'll try to find a workaround, because `serde_json::RawValue` leads to the WASM failed tests:
https://github.com/KomodoPlatform/atomicDEX-API/issues/1042#issuecomment-1348375886

Reply:
Rolled back the changes